x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_trends.ts

```diff
@@ -23,18 +24,30 @@ export interface ScoreTrendDoc {
       failed_findings: number;
     }
   >;
+  score_by_benchmark_id: Record<
+    string,
+    Record<
+      string,
+      {
+        total_findings: number;
+        passed_findings: number;
+        failed_findings: number;
+      }
+    >
+  >;
 }
 
 export type Trends = Array<{
   timestamp: string;
   summary: Stats;
   clusters: Record<string, Stats>;
+  benchmarks: Record<string, Stats>;
 }>;
 
 export const getTrendsQuery = (policyTemplate: PosturePolicyTemplate) => ({
   index: BENCHMARK_SCORE_INDEX_DEFAULT_NS,
-  // large number that should be sufficient for 24 hours considering we write to the score index every 5 minutes
-  size: 999,
+  // Amount of samples of the last 24 hours (accounting that we take a sample every 5 minutes)
+  size: (24 * 60) / 5,
```

Comment:
Let's use please the identical parameter [currently utilized](https://github.com/elastic/kibana/blob/8720a90cb38c0c39c4b0d9229de5a8c6497a46db/x-pack/plugins/cloud_security_posture/server/tasks/findings_stats_task.ts#L38) in the task manager configuration to stay synchronized with any future modifications. It might necessitate confirming that the calculation result is an integer by rounding the outcome.

Reply:
Hey @CohenIdo, this is a score trends aggregation query  which will return the size of the number of trend documents. I don't see how I can apply the 5m interval here.